HolyshiftService/DB models/HolyShiftUser.cs

```diff
@@ -0,0 +1,17 @@
+public class HolyShiftUser
+{
+    public string Id { get; set; }
+    public string UserName { get; set; }
+    public string Email { get; set; }
+    public string PasswordSalt { get; set; } 
+    public string PasswordHash { get; set; } 
+
+    public HolyShiftUser()
```

Comment:
You don't need a constructor here.

You can say

`public string Id { get; set; } = string.Empty;`

> Note: In .net7, they added keyword `required` to the properties so it will help when we migrate to .net 8.